Update flutter_typeahead.dart

valiantv7 opened the pull request:
Added ability to toggle the suggestion box without needing to focus/unfocus the text field. This is useful in case one wants to disable the text field e.g. in order to make it work like a drop down. 

The drop down implementation would look like this:
GestureDetector(
        onTap: () {_suggestionsBoxController.toggleUnfocused();},
        child: TypeAheadFormField<DropDownItem>(
                 textFieldConfiguration: TextFieldConfiguration(
                        enabled: false)))

sjmcdowall commented:
Will need a test in the test directory that passes (no chance you can fix the failing one that should be removed maybe?) -- and could you update the examples in the README on how to use (like your example?)

valiantv7 commented:
I have updated the README and added tests for the SuggestionsBoxController. Regarding the failing test, I don't understand why/how sending arrowDown is supposed to unfocus the text field and focus the first suggestion?
